packages/x-data-grid-pro/src/internals/index.ts

```diff
@@ -1,16 +1,15 @@
-// eslint-disable-next-line import/export
+/* eslint-disable import/export */
```

Comment:
@LukasTy Let me know if this should not be ignored because I see duplicate exports which eslint is complaining about.